notepad/Notepad.js

```diff
@@ -16,6 +16,33 @@ var ImageTool = require('substance/packages/image/ImageTool');
 var MarkTool = require('./MarkTool');
 var TodoTool = require('./TodoTool');
 
+Controller.prototype.uploadFile = function(file, cb) {
```

Comment:
It's not good practice to monkey patch the Controller prototype. Instead you can just add that method to the Notepad prototype. I will do it in the course of my refactor anyway. Just to let you know for next time.